Packs/TrendMicroVisionOne/Integrations/TrendMicroVisionOneV3/TrendMicroVisionOneV3.py

```diff
@@ -2429,20 +2430,19 @@ def update_status(
     workbench_id = args.get(WORKBENCH_ID, EMPTY_STRING)
     status = args.get(STATUS, EMPTY_STRING)
     if_match = args.get(IF_MATCH, EMPTY_STRING)
-    inv_res = args.get(INV_RESULT, None)
-    inv_sts = args.get(INV_STATUS, None)
+    inv_res = args.get(INV_RESULT, EMPTY_STRING)
+    # inv_sts = args.get(INV_STATUS, EMPTY_STRING)    # Deprecated
```

Comment:
Why not delete it? if it's deprecated

Reply:
I have updated the code and removed references to inv_status.